client/trino-jdbc/src/test/java/io/airlift/units/ThreadCount.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.units;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.OptionalInt;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import static io.airlift.units.Preconditions.checkArgument;
+import static java.lang.Math.min;
+import static java.lang.Math.round;
+import static java.lang.Math.toIntExact;
+import static java.lang.String.format;
+import static java.nio.file.Files.exists;
+import static java.nio.file.Files.lines;
+
+// This class is a copy from airlift's units due to an inability to use
+// newer units version in JDBC due to different JDK target.
+// It is temporary solution until client and JDBC are moved to JDK 11+.
+// This class is added to test classes, so it won't be a part of the jdbc driver.
+public class ThreadCount
```

Comment:
Why would we need this copy?
plase remove

Reply:
Jdk 8 target and newer units is on 11

Reply:
i know. but trino-jdbc should avoid having classes in io.airlift

Reply:
It’s the matter of test scope

Reply:
So this class resides in trino-jdbc tests, it is unused there, and must remain API-compatible with the newly added ThreadCount class in units. Does this sounds like we're doing something wrong here?

Reply:
Yeah. Keeping JDBC on JDK 8 which prohibits upgrading dependencies like airlift units

Reply:
If i understand correctly, trino-jdbc ships units 1.7 for its own use and it downgrades units version for test scope resulting in NoClassDefFoundError when we try to start trino server.

unless maven allows us to package units 1.7 but still use a newer version for test classpath, i see the following options:
- stop using airlift units in trino-jdbc (perhaps copying & repackaging necessary classed if this is indeed warranted)
- move jdbc tests that require trino server to a separate module, where we will use modern units.

Reply:
Moving tests seems like a better option